include/vicon_estimator.h

```diff
@@ -33,6 +33,9 @@ static const double kDefaultDt = 0.01;
 // values
 static const double kDefaultTranslationalKp = 1.0;
 static const double kDefaultTranslationalKv = 10.0;
+static const double kDefaultOutlierThresholdM = 0.5;
```

Comment:
Or alternatively, and probably better, say kDefaultOutlierThresholdMeters. The trailing letters are used to indicate frames in this package.